starter-dev-e2e/cypress/integration/dashboard.spec.ts

```diff
@@ -41,10 +65,27 @@ describe("Sign in", () => {
     ).as("restUserTopRepos");
   });
 
-  describe("My First Test", () => {
-    it("top repos should be listed", () => {
-      //TODO: In #264, we should properly look for elements using `[data-testid="some-test-id"]`
-      cy.get("h2").should("be.visible");
+  it("should display list of gists", () => {
+    cy.get(`[data-testid="show gists list"]`).should("be.visible");
+    cy.wait("@gqlCurrentUserQuery")
+    .wait("@gqlUserGistsQuery");
+    cy.contains('gist example 1').should('have.attr', 'href')
+    .then(href => {
+      expect(href).to.match(/https:\/\/gist\.github\.com/)
     });
   });
+
+  it("top repos should be listed with valid repo links", () => {
+    cy.get(`[data-testid="show repo list"]`).should(`be.visible`);
+    cy.contains("starter.dev-github-showcases").click();
+    cy.wait("@gqlRepoPageQuery")
+      .wait("@gqlRepoTreeQuery")
+      .wait("@gqlRepoReadMeQuery");
+    cy.url().should("include", "/thisdot/starter.dev-github-showcases");
+    cy.get(`[data-testid="readme"]`).should("be.visible");
+  });
+
+  it("should be able to see user's avatar in header", () => {
+    cy.get(`[data-testid="user avatar header"]`).should(`be.visible`);
```

Comment:
we should probably additionally check that the avatar contains the avatar returned in the response:

```
      .and("have.attr", "src")
      .and(
        "match",
        /.*(https).*(avatars\.githubusercontent\.com).*(22839396).*/
      )
```

Ensure the `22839396` is the correct value returned by the response, but I believe it should be -- this regex will work against both `angular-apollo` and `next-react`!